test/stripe/transfer_test.rb

```diff
@@ -84,5 +84,16 @@ class TransferTest < Test::Unit::TestCase
         assert reversals.data.is_a?(Array)
       end
     end
+
+    should "retrieve a reversal with expand" do
+      reversal = Stripe::Transfer.retrieve_reversal(
+        "tr_123",
+        "trr_123",
+        { expand: %w[transfer] },
+        {}
+      )
+      assert_requested :get, "#{Stripe.api_base}/v1/transfers/tr_123/reversals/trr_123?expand%5B%5D=transfer"
+      assert reversal.is_a?(Stripe::Reversal)
+    end
```

Comment:
Nice! Can we add a few more tests here though, similar to https://github.com/stripe/stripe-ruby/pull/1294/files#diff-69a0403918bdeaa82349423a9f4508ab63ec9437d95ee36fa9b7271f527c6482, to check that it still works with just opts, throws an error if only params are passed in, etc.?